apps/admin/src/containers/Purchase/index.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react';
+import { Form, Header, Confirm, Select } from 'semantic-ui-react';
+import { ModalOperation } from '@thecointech/shared/containers/ModalOperation';
+import { DualFxInput } from '@thecointech/shared/components/DualFxInput';
+import { UxAddress } from '@thecointech/shared/components/UxAddress';
+import { toHuman } from '@thecointech/utilities';
+import { BuyAction, getActionFromInitial, PurchaseType } from '@thecointech/broker-db';
+import { Processor } from '@thecointech/tx-processing/deposit';
+import { DateTime } from 'luxon';
+import Decimal from 'decimal.js-light';
+import messages from './messages';
+import { getCurrentState } from '@thecointech/tx-processing/statemachine';
+import { AccountMap } from '@thecointech/shared/containers/AccountMap';
+import { Account } from '@thecointech/shared/containers/Account';
+const typeOptions = ["deposit", "other"].map(k => ({
+  key: k,
+  value: k,
+  text: k
+}));
+
+export const Purchase = () => {
+
+  const [purchaser, setPurchaser] = React.useState("0x1234");
+  const [forceValidate, setForceValidate] = React.useState(false);
+  const [type, setType] = React.useState<PurchaseType>("deposit");
+  const [fiat, setFiat] = React.useState(0);
+  const [confirm, setConfirm] = React.useState(false);
+  const [isProcessing, setIsProcessing] = React.useState(false);
+  const [step, setStep] = React.useState("");
+
+  const account = AccountMap.useActive()!;
+  const api = Account(account.address).useApi();
+
+  const onConfirm = async () => {
+    setConfirm(false);
+    setIsProcessing(true);
+    setForceValidate(true);
+    throw new Error("Just because it compiles, doesn't mean it works.  Check properly");
```

Comment:
This class is for manual purchases.  It probably won't ever gt used again

Reply:
Should it be named "obsolete" of "legacy"? 😄

Reply:
Joke aside we could put a log line in here and check in a while if it's used at all.

Reply:
It runs in Admin.  The only person who see's it is me (for the foreseeable future).

After thinking about it, it will probably be resurrected at some point for non-ETransfer based transfers.  Which will (hopefully) happen

Reply:
But that's a good point: better than just 'throwing' (which is quite unexpected).  I'll disable the button & change the text to make it clear this component is unusable